Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -1368,7 +1400,7 @@ Atom *ParseMolFileAtomLine(const std::string text, RDGeom::Point3D &pos,
       massDiff = FileParserUtils::toInt(text.substr(34, 2), true);
     } catch (boost::bad_lexical_cast &) {
       std::ostringstream errout;
-      errout << "Cannot convert '" << text.substr(34, 2) << "' to into on line "
+      errout << "1Cannot convert '" << text.substr(34, 2) << "' to int on line "
```

Comment:
Did you intend to leave the numbering in these error strings?

Reply:
fixed